internal/config.go

```diff
@@ -4,5 +4,5 @@ var (
 	// Experimental enables experimental features if set to "on"
 	Experimental = "on"
 	// MetadataVersion defines the current schema version
-	MetadataVersion = "v0.1"
+	MetadataVersion = "v0.2"
```

Comment:
Do we want to update the metadata version ? I dont't think we should until we publish an actual CNAB-based release). WDYT @silvin-lubecki

Reply:
Well, as removing the namespace breaks the validation of current packages, I think it's better to bump the json schema version. And this new version will be released in the next docker-app release, which will include all the CNAB work.

Reply:
@chris-crone